coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -198,6 +226,11 @@ def from_metadata(cls, doccomment, docstyle_definition,
                                   section.name +
                                   docstyle_definition.metadata.param_end)
 
+            elif isinstance(section, cls.ExceptionValue):
+                assembled_doc += (docstyle_definition.metadata.exception_start
+                                  + section.name
```

Comment:
yay for using `+` after a line break, whereas the previous branch used `+` before the line break.

your code is correct (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

That part of pep 8 was modified about a year ago, iirc.